test/parallel/test-process-env.js

```diff
@@ -67,3 +67,12 @@ assert.equal(3, date.getUTCHours());
 assert.equal(5, date.getHours());
 */
 /* eslint-enable max-len */
+
+// environment variables should be case-insensitive on Windows, and
+// case-sensitive on other platforms
+process.env.TEST = 'test';
+if (common.isWindows) {
+  assert.strictEqual(process.env.TEST, process.env.teST);
+} else {
+  assert.strictEqual('undefined', typeof process.env.teST);
```

Comment:
Instead of checking the `typeof`, can you just directly compare against `undefined`.